### ISSUE TITLE
BUG: RuntimeError: an error occurred during databases indexation see formatdb.log

### ISSUE THREAD

**Issue by linnabrown** [dcouvin/CRISPRCasFinder#4]
```
Sequence number 34..Traceback (most recent call last):
  File "/home/ciillab/huangle/CRISPRCasFinder/bin/macsyfinder", line 329, in <module>
    idx.build(force = config.build_indexes)
  File "/home/ciillab/huangle/CRISPRCasFinder/macsyfinder-1.0.5/macsypy/database.py", line 118, in build
    raise RuntimeError(msg)
RuntimeError: an error occurred during databases indexation see formatdb.log

```

**Comment by bneron** [comment]
I think it's because your data are in a directory where you are not allow to write in.
MacSyFinder try to create index in the same directory as the data. so this directory must be writeable.
we will change this behaviour in the future but with the current version we cannot change it.

**Comment by dcouvin** [comment]
Thank you for your messages. The RuntimeError has been fixed in the recently updated version 4.2.19 of CRISPRCasFinder. Other changes will be made in the future.
